<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>380</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Загрузка</string>
    </property>
    <widget class="QComboBox" name="viewCombo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="viewLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Представление модели    </string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>380</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Камеры</string>
    </property>
    <widget class="QDoubleSpinBox" name="cameraXSpin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="cameraYSpin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="cameraZSpin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="camerasList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>360</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addCameraButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setCameraButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Установить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>370</y>
      <width>380</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Объекты</string>
    </property>
    <widget class="QListWidget" name="objectsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>360</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="removeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить выделенное</string>
     </property>
    </widget>
    <widget class="QPushButton" name="compositeButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Скомпоновать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>620</y>
      <width>380</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>Трансформирование</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>dz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>dx</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="moveXSpin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="moveYSpin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="moveZSpin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>dy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="moveButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Перенос</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>kx</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="scaleButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Масштаб</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scaleXSpin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scaleYSpin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scaleZSpin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>ky</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>kz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>ox</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Поворот</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotateXSpin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotateYSpin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="rotateZSpin">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>oy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>oz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
